tests/data_context/test_data_context_config_errors.py

```diff
@@ -32,19 +33,19 @@ def _build_local_dir(fixture_subdir: str | None = None) -> pathlib.Path:
 def test_DataContext_raises_error_on_config_not_found(build_local_dir: Callable):
     local_dir = build_local_dir()
     with pytest.raises(gx_exceptions.ConfigNotFoundError):
-        get_context(context_root_dir=local_dir)
+        DataContext(context_root_dir=local_dir)
```

Comment:
All these tests specify `DataContext` - although it is outdated, we should be testing that method and not `get_context`